pkg/crypto.go

```diff
@@ -70,5 +79,20 @@ func (c Crypto) Setup(hmacFile string, masterFile string) (bool, error) {
 }
 
 func (c Crypto) Protect(value string) (string, error) {
-	return value, nil // TODO implement this
+	log.Infof("%s > Protecting text using crypto.", c.instance.ID())
+	requestFormData := c.instance.http.RequestFormData(map[string]any{"datum": value})
```

Comment:
inline 83 & 84 line (remove `requestFormData` var

Reply:
extra vars are fine when they are reused or introduce better readability; I don't see any of these